lib/uniris/oracles/oracles/uco_price.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Uniris.Oracles.UcoPrice do
```

Comment:
I think you should you put this file into `oracles/uco_price.ex` instead of `oracles/oracles/uco_price.ex` because the double `oracles` folder brings confusion.